daana-Banerjeed/hw_1/99bottles.js

```diff
@@ -0,0 +1,23 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+*/
+
+for (i = 99; i >= 0; i--) {
+  console.log(i + ((i != 1) ? " bottles" : " bottle") + " of beer on the wall, ")
+  console.log(i + ((i != 1) ? " bottles" : " bottle") + " of beer. " +
+  "Take one down, pass it around... ");
+}
```

Comment:
We rely on conditionals to determine whether to print one thing or another. These conditionals should be revisited each iteration (or each loop) to print out the correct lyric, otherwise we lose control of what makes if/else conditionals inside loops so powerful.

It also helps to be explicit about what condition we're seeking, so more or less than a specified value; `!drink` could be anything - but we're looking for a condition to stop our loop, and counting down or up to that helps others understand your intention.

Keep going above and beyond by digging into JavaScript's vanilla methods. This is really exciting to see, Daana.